cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -389,6 +389,52 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    SelfPacedDueDateEditor = AbstractEditor.extend({
+        fieldName: 'due_num_weeks',
+        templateName: 'self-paced-due-date-editor',
+        className: 'modal-section-content has-actions due-date-input grading-due-date',
+
+        events: {
+            'click .clear-date': 'clearValue',
+            'keyup #due_in': 'validateDueIn'
```

Comment:
Maybe more of a question for @ciduarte , but should this be a `keyup` event or a `blur` or perhaps something else altogether?